src/Dictionaries/DictionaryStructure.cpp

```diff
@@ -241,6 +241,8 @@ const DictionaryAttribute & DictionaryStructure::getAttribute(const std::string
         for (const auto & key_attribute : *key)
             if (key_attribute.name == attribute_name)
                 return key_attribute;
+
+        throw Exception{"No such attribute '" + attribute_name + "'", ErrorCodes::BAD_ARGUMENTS};
```

Comment:
Please information that this attribute is related to attributes is from keys attributes. 
Currently this necessary only for IP trie but will be easier to debug.